subprojects/stdlib/meson.build

```diff
@@ -0,0 +1,14 @@
+project('stdlib', 'c')
+
+extern_library_helper_dep = subproject('vinumc').get_variable('extern_library_helper_dep')
+
+stdlib_srcs = [
+  'stdlib.c',
+]
+
+stdlib = shared_library(
+  'stdlib',
+  stdlib_srcs,
+  dependencies : extern_library_helper_dep,
```

Comment:
Oh, now I see why you create the shared lib. You can create the dep object with only the header file, so you don't need to create a new shared lib.

```meson
my_dependency = declare_dependency(
  include_directories: include_directories('path/to/header')
)

```

Reply:
Done.